src/old/v1/newTheme.ts

```diff
@@ -17,7 +17,7 @@ const defaultTheme = {
       borderHoverColor: '#FFF'
     },
     secondary: {
-      background: '#FFF',
+      background: 'transparent',
```

Comment:
does this match the new button style nejc created too? if not it should be updated there as this button will soon be legacy

Reply:
new buttons looks formatted a bit differently so i don't know if this will be an issue there, if it is i can re-address it then. i kinda figured this would get tossed in the shuffle.